apps/api/src/app/shared/services/logger.service.ts

```diff
@@ -0,0 +1,14 @@
+import { createNestLoggingModuleOptions, PinoLogger } from '@novu/application-generic';
+import packageJson from '../../../../package.json';
+
+export const getLogger = (context: string) => {
```

Comment:
shouldn't this come from the application generic so we can reuse in the worker if needed?

Reply:
will definitely move it once we do the same to the apps, for now i kept it close.